### ISSUE TITLE
plot data in eps_fit_lorentzian.py

### ISSUE THREAD

**Issue by htakamur** [NanoComp/meep#2303]
https://github.com/NanoComp/meep/blob/fdd5e0e47faa6044f97a0c11b019f683d6fd122e/python/examples/eps_fit_lorentzian.py#L146

To compare fit and actual data visually in a plot, "+ eps_inf" seems to be required for the actual epsilon. 

```
146c146
< plt.plot(wl_reduced, np.real(eps_reduced), "bo-", label="actual")
---
> plt.plot(wl_reduced, np.real(eps_reduced) + eps_inf, "bo-", label="actual")

```

**Comment by oskooi** [comment]
Good catch. Feel free to submit a PR with this fix.

**Comment by oskooi** [comment]
Fixed by #2304.
